data/mods/Mundane_Zombies/modinfo.json

```diff
@@ -3,45 +3,68 @@
     "type": "MOD_INFO",
     "ident": "Mundane_Zombies",
     "name": "Mundane Zombies",
-    "authors": [ "daftfad" ],
+    "authors": [ "Theundyingcode", "daftfad" ],
+	"maintainers": [ "Theundyingcode" ],
```

Comment:
Indentation.

Reply:
odd, it looks correct in notepad++

Reply:
or are you saying the maintainers line should be indented because none of the other mods I opened are like that?

Reply:
No, I say it should on one vertical line as other fields.